scalyr_agent/builtin_monitors/windows_event_log_monitor.py

```diff
@@ -232,7 +246,29 @@ def __init__( self, config, logger, channels ):
 
         self.__bookmarks = {}
 
-    def load_checkpoints( self, checkpoints ):
+    def load_checkpoints( self, checkpoints, config ):
+
+        # see if we need to create a remote connection
+        if self._server is not None and self._server != 'localhost':
+            username = config.get( 'remote_user' )
+            password = config.get( 'remote_password' )
+            domain = config.get( 'remote_domain' )
+            flags = win32evtlog.EvtRpcLoginAuthDefault
+
+            # login object is a tuple
+            login = (self._server, username, domain, password, flags )
+            self._logger.log( scalyr_logging.DEBUG_LEVEL_0, "Performing remote login: server - %s, user - %s, domain - %s" % (self._server, username, domain) )
```

Comment:
DEBUG_LEVEL_0 is the same level as INFO... let's do DEBUG_LEVEL_1 here.

Reply:
Done.